jarbas/core/management/commands/reimbursements.py

```diff
@@ -77,16 +81,36 @@ def serialize(self, reimbursement):
 
         return reimbursement
 
-    def create_or_update(self, reimbursements_as_dicts):
-        for count, reimbursement in enumerate(reimbursements_as_dicts):
+    def create_or_update(self, reimbursements_as_dicts, status):
+        for reimbursement in reimbursements_as_dicts:
             document_id = reimbursement.get('document_id')
-            if document_id:
-                Reimbursement.objects.update_or_create(
-                    document_id=document_id,
-                    defaults=reimbursement
-                )
-            self.print_count(Reimbursement, count=count + 1)
+
+            if not document_id:
+                self.count['skip'] += 1
+                status.change(self.status)
+                continue
+
+            _, created = Reimbursement.objects.update_or_create(
+                document_id=document_id,
+                defaults=reimbursement
+            )
+
+            key = 'created' if created else 'updated'
+            self.count[key] += 1
+            status.change(self.status)
 
     def mark_not_updated_reimbursements(self):
         qs = Reimbursement.objects.filter(last_update__lt=self.started_at)
         qs.update(available_in_latest_dataset=False)
+
+    @property
+    def status(self):
+        label = '{}s'.format(self.get_model_name(Reimbursement)).lower()
+        total = sum(map(self.count.get, self.count.keys()))
+        output = [
+            'Processed: {} lines'.format(total),
```

Comment:
Hum… I coded it that way because I wasn't considering the CSV line a reimbursement until the [_has a `document_id`_ check](https://github.com/datasciencebr/jarbas/blob/3b9a0b923159187ce52f0068be7d4ddc780889a8/jarbas/core/management/commands/reimbursements.py#L88-L91). Not sure if it makes sense — how do you prefer?